src/harness/sas_test_harness.py

```diff
@@ -345,7 +345,7 @@ def generateCbsdRecords(registration_requests, grant_requests_list):
                                    registration_request['cbsdSerialNumber'])
 
     internal_data = {
-        'id': 'cbsd/{}'.format(cbsd_reference_id),
+        'id': '{}'.format(cbsd_reference_id),
```

Comment:
Can't you just use `cbsd_reference_id` directly?

Reply:
yes, you are right. changed as suggested.